tests/test_mail.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+from fastapi.testclient import TestClient
+from fastapi_mail import MessageSchema
+from pydantic import ValidationError, EmailStr
+
+import config
+from app.main import app
+from internal.mail import send_fast_email
+
+pytest_plugins = "smtpdfix"
+
+client = TestClient(app)
+
+
+@pytest.fixture
+def configured_smtpd(smtpd):
+    """
+    Overrides the SMTP related configuration to use a mock SMTP server
+    :param smtpd: the smtpdfix fixture that represents an SMTP server
+    :return: smtpd
+    """
+
+    def override_settings():
+        return config.Settings(smtp_server=smtpd.hostname,
+                               smtp_port=smtpd.port)
+
+    app.dependency_overrides[config.get_settings] = override_settings
+    yield smtpd
+    app.dependency_overrides.pop(config.get_settings)
+
+
+def assert_validation_error_missing_body_fields(validation_msg,
+                                                missing_fields):
+    """
+    helper function for asserting with open api validation errors
+    look at https://fastapi.tiangolo.com/tutorial/path-params/#data-validation
+    :param validation_msg: the response message after json
+    :param missing_fields: a list of fields that are asserted missing
+    """
+    assert isinstance(validation_msg, dict)
+    assert 1 == len(validation_msg)
+    assert "detail" in validation_msg
+    details = validation_msg["detail"]
+    assert isinstance(details, list)
+    assert len(missing_fields) == len(details)
+    for detail in details:
+        assert 3 == len(detail)
+        assert "type" in detail
+        assert "value_error.missing" == detail["type"]
+        assert "msg" in detail
+        assert "field required" == detail["msg"]
+        assert "loc" in detail
+        loc = detail["loc"]
+        assert isinstance(loc, list)
+        assert 2 == len(loc)
+        assert "body" == loc[0]
+        assert loc[1] in missing_fields
+
+
+def test_read_main():
+    response = client.get("/")
+    assert response.status_code == 200
+
+
+def test_send_mail_no_body(configured_smtpd):
+    response = client.post("/mail/invitation/")
+    assert response.status_code == 422
```

Comment:
Prefer `status.HTTP_422_UNPROCESSABLE_ENTITY` (read more about it [here](https://fastapi.tiangolo.com/tutorial/handling-errors/))

Reply:
ok :)